cmd/axelard/cmd/vald/start.go

```diff
@@ -74,6 +74,13 @@ func GetValdCommand() *cobra.Command {
 			logger := serverCtx.Logger.With("module", "vald")
 			v := serverCtx.Viper
 
+			if err := v.BindPFlag("tss.tofnd-host", cmd.PersistentFlags().Lookup("tofnd-host")); err != nil {
+				return err
+			}
+			if err := v.BindPFlag("tss.tofnd-port", cmd.PersistentFlags().Lookup("tofnd-port")); err != nil {
+				return err
+			}
+
```

Comment:
```suggestion

			if err := v.BindPFlag("tss.tofnd-port", cmd.PersistentFlags().Lookup("tofnd-port")); err != nil {
				return err
			}

			if err := v.BindPFlag("tss.tofnd-dial-timeout", cmd.PersistentFlags().Lookup("tofnd-dial-timeout")); err != nil {
				return err
			}

```

Reply:
done, although I had to declare`tofnd-dial-timeout` as a new flag for the command

Reply:
Ah, I see, didn't realize that wasn't a flag. Might remove it in a refactor since it might be confusing as a flag